docs/architecture/2026-07-12-agent-architecture-audit.html

```
@@ -0,0 +1,111 @@
+<title>PTSO Agent — Architecture Audit</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)